### ISSUE TITLE
Long text in views' abstract overflows if longer than given column in Chrome

### ISSUE THREAD

**Issue by antobenve** [unep-grid/map-x-mgl#350]
![image](https://user-images.githubusercontent.com/30286565/47779111-9daa8b80-dcf8-11e8-9497-43cc6d1424c8.png)

**Comment by antobenve** [comment]
I reopen this issue as it still occurs in Chrome while in Firefox the text is split in two lines and the error is not generated.
The problem occurs when the text is too long and is not strictly related to urls (normal text overflows as well).

chrome
![image](https://user-images.githubusercontent.com/30286565/51050067-e5d70b80-15cf-11e9-82c5-31cbd52cd0c9.png)
firefox
![image](https://user-images.githubusercontent.com/30286565/51050100-f8514500-15cf-11e9-9b8f-55d8bf5fdadf.png)

Tested in local v.1.5.37 Windows 10 Firefox (63.0.3 (64-bit)) and Chrome (71.0.3578.98 (64-bit))

**Comment by thomaspiller** [comment]
This seems to be fixed in v1.8.3-beta. @antobenve Can you test on your side and close the issue if that's the case? Thanks

**Comment by antobenve** [comment]
Long text associated to URLs is not broken in different lines in Chrome (in Firefox it is). However, with the new abstract format (1 column only) the issue results in text being cut at the edge of the field rather than extending beyond it. I think it is a good strategy based on how Chrome handles this, so I suggest we can close the issue.

**Comment by thomaspiller** [comment]
After having tested again, I confirm that the problem is still present.
One solution would be to add [word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) to the css. This is supported by nearly [98% of users](https://caniuse.com/?search=word-break).

```css
word-break: break-all;
```
Another solution would be to use [overflow-wrap](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) supported by [99% of users](https://caniuse.com/?search=overflow-wrap).

```css
overflow-wrap: break-word OR anywhere;
```

@fxi Would it be possible to take a look and fix it before deploying the next version to production?

**Comment by thomaspiller** [comment]
Fixed in 1.8.4-beta. Thanks.
